inst/NORIC_local_monthly_stent.Rmd

```diff
@@ -12,7 +12,7 @@ header-includes:
   - \usepackage{booktabs}
   - \usepackage{rotating}
```

Comment:
Her må du legge til ```- \usepackage{float}``` for at 'H' opsjonen skal fungere i latex (tilsvarende NORIC_local_monthly.Rmd)